src/main/resources/META-INF/rewrite/spring-framework-70.yml

```
@@ -27,3 +27,9 @@ recipeList:
       newVersion: 7.0.x
   - org.openrewrite.java.testing.junit6.JUnit5to6Migration
   - org.openrewrite.java.jackson.UpgradeJackson_2_3
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: 'org.springframework.lang.Nullable'
+      newFullyQualifiedTypeName: 'org.jspecify.annotations.Nullable'
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: 'org.springframework.lang.NonNull'
+      newFullyQualifiedTypeName: 'org.jspecify.annotations.NonNull'
```

Might make sense to use the existing recipe we have here:
https://docs.openrewrite.org/recipes/java/jspecify/migratefromspringframeworkannotations
```suggestion
  - org.openrewrite.java.jspecify.MigrateFromSpringFrameworkAnnotations
```
You'll want to add a dependency on rewrite-jackson here as well.